src/chainstate/stacks/db/mod.rs

```diff
@@ -132,6 +132,7 @@ pub struct StacksAccount {
 #[derive(Debug, Clone, PartialEq)]
 pub struct MinerPaymentSchedule {
     pub address: StacksAddress,
+    pub recipient_contract: Option<QualifiedContractIdentifier>,
```

Comment:
Is there any use for specifying a different `StacksAddress` as the recipient, rather than a contract? As in, what if miner Alice wanted coinbase rewards to go to Bob even if Bob isn't a contract?

Reply:
I thought about this too, but I can't think of a compelling use-case.  If Alice wanted to pay Bob the block reward, then why can't Alice just fund Bob's block-commit transaction?

Reply:
Because they don't share keys -- like one situation could be that the coinbase rewards are going to a "cold" Stacks wallet, which you wouldn't want to use as the miner address (because the miner address must be hot).

Reply:
Gotcha.  I've made it so that the coinbase can pay to either a contract or a standard principal.